sound/soc/sof/intel/hda.h

```diff
@@ -17,9 +17,13 @@
 #define PCI_TCSEL			0x44
 #define PCI_CGCTL			0x48
 
```

Comment:
better to correct naming of these 2 registers:
'''
#define PCI_PGCTL			0x44
#define PCI_CGCTL			0x48